libqtile/widget/check_updates.py

```diff
@@ -38,7 +38,8 @@ class CheckUpdates(base.ThreadedPollText):
         ("colour_no_updates", "ffffff", "Colour when there's no updates."),
         ("colour_have_updates", "ffffff", "Colour when there are updates."),
         ("restart_indicator", "", "Indicator to represent reboot is required. (Ubuntu only)"),
-        ("no_update_string", "", "String to display if no updates available")
+        ("no_update_string", "", "String to display if no updates available"),
+        ("refresh_after_execute", True, "Refresh update count after execute")
```

Comment:
Can we drop this config option? It's not clear to me when you'd ever not want to do this.